### ISSUE TITLE
Does this no longer work with OAuth for Apps deprecated?

### ISSUE THREAD

**Issue by Joseph-Zaino** [home-assistant/home-assistant.github.io#26015]
### Feedback

You say, "Note that OAuth for Apps has been deprecated by Google and will break by October 2022." 

So, does the instruction on this page no longer work?

### URL

https://www.home-assistant.io/integrations/nest/

### Version

2023.1.7

**Comment by allenporter** [comment]
No the instructions work fine. It's talking about a specific type of authentication that used to be the primary way to authenticate.

**Comment by allenporter** [comment]
(there is an alert at the top of the page)

**Comment by Joseph-Zaino** [comment]
Okay. It would be good if the instruction included that the current
instructions work, and OAuth for Apps no longer works. I almost stopped
trying after reading that line.

I got it working as I suspected the current instructions would work. Thank
you for your work on this project!

**Comment by allenporter** [comment]
I welcome contributions to improve the documentation if it's hard to follow. Thanks!
